### DISCUSSION TITLE
Article on running Conductor 3 with Elasticsearch 7 and Postgres

### DISCUSSION THREAD

**Discussion by erikbrgr** [Netflix/conductor#2858] [General]
Hi all! I recently wrote an article on how to get Conductor running with Elasticsearch 7 and Postgres. I was wondering if anyone here might find it useful.

It is using Conductor 3.5.1. It was published just before 3.6 came out, so I might update it in the near future. But it should be easy to apply the ideas the 3.6. I hope :)

You can find the article here: https://betterprogramming.pub/running-netflix-conductor-3-in-docker-using-elasticsearch-7-and-postgresql-b415988dd74a

I hope this isn't against the rules, if it is, please let me know or remove this post.

**Comment by aravindanr** [comment]
> It is using Conductor 3.5.1. It was published just before 3.6 came out, so I might update it in the near future. But it should be easy to apply the ideas the 3.6. I hope :)

3.6.0 includes the workflow editor in the UI, but there are no major changes in the server. The steps in the blog post should work with 3.6.0 as well.
